packages/flutter_tools/lib/src/drive/drive_service.dart

```diff
@@ -262,7 +262,8 @@ class FlutterDriverService extends DriverService {
     try {
       final int result = await _processUtils.stream(<String>[
         _dartSdkPath,
-        ...<String>[...arguments, testFile, '-rexpanded'],
+        ...arguments, 
```

Comment:
what about `-rexpanded`?

Reply:
> what about `-rexpanded`?

> > You'll notice I also removed `-rexpanded`; @jonahwilliams believes that was accidentally copied over from else-where, and never did anything (someone I guess could have parsed it in `void main(...)`, but given https://github.com/flutter/flutter/issues/51135 & https://github.com/flutter/flutter/issues/145499 that seems unlikely.

Reply:
(We could do some gymnastics to retain that argument if we wanted to, but I don't see a reason to?)

Reply:
Yeah, since its just a dart script the command line argument goes no where (well it went to args, but what are you gonna do with it?)

Reply:
yeah, i see what you mean. i like -rexpanded, but I can add that later in our /dev/bots dir.

Reply:
> yeah, i see what you mean. i like -rexpanded, but I can add that later in our /dev/bots dir.

I also think, and might be wrong, that `--reporter=expanded` is the default when `dart test` runs on CI (https://pub.dev/packages/test#selecting-a-test-reporter), and if not we can definitely make it _our_ default.

Reply:
we don't run on github actions (although i guess we could set that variable :) )